.github/workflows/ci.yml

```
@@ -11,175 +11,229 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  test-desktop:
-    name: Build and test
+  test:
+    name: Test ${{ matrix.crate }}
     runs-on: ubuntu-latest
+    needs: gather_published_crates
     strategy:
+      fail-fast: false
       matrix:
-        args: [
-          "--no-default-features",
-          "--all-features",
-          "--benches --all-features",
-        ]
+        crate: ${{ fromJSON(needs.gather_published_crates.outputs.members) }}
     steps:
-    - name: Install Protoc
-      uses: arduino/setup-protoc@v1
+      - name: Install Protoc
+        uses: arduino/setup-protoc@v1
```

If we want to pin to the exact sha, it'd be:
```suggestion
        uses: arduino/setup-protoc@64c0c85d18e984422218383b81c52f8b077404d3 # v1.1.2
```